routes/wishlist.routes.js

```diff
@@ -15,9 +15,9 @@ router.post(
   wishlistController.createWishlistItem,
 );
 router.get(
-  '/',
+  '/getitem/:id',
```

Comment:
```suggestion
  '/get-item/:id',
```

it will improve readability :)

Reply:
done